### DISCUSSION TITLE
Unable to use SukiUI

### DISCUSSION THREAD

**Discussion by Pratham95VP** [kikipoulet/SukiUI#215] [Q&A]
I am new to SukiUI. I am following https://kikipoulet.github.io/SukiUI/documentation/ this documentation but not able apply SukiUI theme.  I have attached image. can someone check what have done wrong ? I have used SukiUI nuget package.
![Untitled](https://github.com/kikipoulet/SukiUI/assets/154887456/b559f90b-138f-4bf5-9917-9373c17175e2)

**Errors: 1. The type or namespace name 'SukiWindow' could not be found (are you missing a using directive or an assembly reference?)
2.Unable to resolve type SukiTheme from namespace clr-namespace:SukiUI;assembly=SukiUI Line 9, position 4
3.Unable to resolve type SukiWindow from namespace clr-namespace:SukiUI.Controls;assembly=SukiUI Line 1, position 2**

**Comment by AuroraZiling** [comment]
Check this

https://kikipoulet.github.io/SukiUI/documentation/getting-started/installation.html

**Reply by Pratham95VP** [reply]
> https://kikipoulet.github.io/SukiUI/documentation/getting-started/installation.html

this is the same link I am referring.  The documentation is for Avalonia version was 11.1.0-beta2 with SukiUI version 6.0.0-beta6. I am using Avalonia version was 11.0.10 with SukiUI version 5.3.0 , may be that's what causing the issue. Is there any documentation for SukiUI version 5.3.0 ?

**Reply by AuroraZiling** [reply]
The old documentation (wiki) doesn't have version 5.3.0 either.

It might be possible to clone the repository and then checkout to an much older commit, the demo app from that time are available for reference

**Reply by kikipoulet** [reply]
Or simply use SukiUI 6.0.0-beta6. We should remove indications to 5.3, 6-beta is more stable anyway

**Reply by Pratham95VP** [reply]
I've successfully upgraded Avalonia and SukiUI to the latest beta version, and everything is functioning well. However, I still believe that there should be documentation available for versions that are not in beta, regardless of their stability. I really appreciate what SukiUI offers. It surpasses other alternatives in functionality and usability. Cheers to that!

**Reply by kikipoulet** [reply]
> I've successfully upgraded Avalonia and SukiUI to the latest beta version, and everything is functioning well. However, I still believe that there should be documentation available for versions that are not in beta, regardless of their stability. I really appreciate what SukiUI offers. It surpasses other alternatives in functionality and usability. Cheers to that!

Thanks for your kind words !

Note that @AuroraZiling is working on a documentation website ( https://kikipoulet.github.io/SukiUI/ ) that will really help in the future.
